packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -524,16 +531,18 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     rtl: boolean,
     getToggleButtonProps: (options?: GetToggleButtonPropsOptions) => any,
   ): JSX.Element {
-    const { triggerButton } = this.props
+    const { labelId, triggerButton } = this.props
     const content = this.getSelectedItemAsString(this.state.value)
+    const triggerButtonId = triggerButton['id'] || this.defaultTriggerButtonId
 
     const triggerButtonProps = getToggleButtonProps({
       onFocus: this.handleTriggerButtonOrListFocus,
       onBlur: this.handleTriggerButtonBlur,
       onKeyDown: e => {
         this.handleTriggerButtonKeyDown(e, rtl)
       },
-      'aria-label': content,
+      'aria-label': undefined,
+      'aria-labelledby': `${labelId} ${triggerButtonId}`,
```

Comment:
so the trigger button is labeled by the label id and himself? 😕 
Also, why wasn't `'aria-labelledby': triggerButtonId` even before this change?

Reply:
Also, we don't handle this case for the trigger input at all? Inputs don't have `aria-labelledby`?

Reply:
1. That's what ARIA says. https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html
1.1 Because we used `aria-label` instead. Now we've corrected it.
2. Downshift handles that by default. That is why it is passed to it as prop.

Reply:
thanks for clarifying